Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# History Error Fix

## Developer changelog:
- Fixed all `history.cpp:1059` errors.


# How to test:
First, you have to copy `Warcraft-Guardians-of-Azeroth-2.mod` from `Crusader Kings III\mod\Warcraft-Guardians-of-Azeroth-2` outside, to `Crusader Kings III\mod` with replace.

Open `Crusader Kings III\logs\error.log`. Confirm there are no `history.cpp:1059` errors.
